scripts/get-helm-3

```diff
@@ -206,7 +206,7 @@ verifySignatures() {
     gpg_stderr_device="/dev/stderr"
   fi
   gpg --batch --quiet --homedir="${gpg_homedir}" --import "${HELM_TMP_ROOT}/${keys_filename}" 2> "${gpg_stderr_device}"
-  gpg --batch --no-default-keyring --keyring "${gpg_homedir}/pubring.kbx" --export > "${gpg_keyring}"
+  gpg --batch --no-default-keyring --keyring "${gpg_homedir}/pubring.gpg" --export > "${gpg_keyring}"
```

Comment:
Can you make this configurable? When I was testing on new Ubuntu, it was `.kbx`.

Maybe at top of file:
```
: ${GPG_PUBRING:="pubring.kbx"}
```

Then you can set `GPG_PUBRING=pubring.gpg` (or vice versa)

Reply:
Thanks and done